AElfContractDecompiler/Models/DecoderConstants.cs

```diff
@@ -0,0 +1,8 @@
+namespace AElfContractDecompiler.Models
+{
+    public static class DecoderConstants
+    {
+        public const string OutPathByDll = @"C:\\Xxx\\OutPathByDll"; // directory of output
```

Comment:
remove hardcode, try use path like this:
_appDataPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), ApplicationFolderName);